sstables/compress.cc

```diff
@@ -264,7 +264,7 @@ local_compression::local_compression(const compression& c)
         sstring n(c.name.value.begin(), c.name.value.end());
         return compressor::create(n, [&c, &n](const sstring& key) -> compressor::opt_string {
             if (key == compression_parameters::CHUNK_LENGTH_KB || key == compression_parameters::CHUNK_LENGTH_KB_ERR) {
-                return to_sstring(c.chunk_len);
+                return to_sstring(c.chunk_len / 1024);
             }
```

Comment:
What do you mean that no compressors implement it? I think all compressors respect chunk_length_in_kb, no?

Reply:
chunk_length_in_kb is indeed always used in the higher layer - the code which uses the chosen compressor to compress chunks of sstable data. But none of the compressors in their code cared about chunk_length_in_kb until now. The other compressors don't assume anything about the length of the input they get each time `compress` is called. In Zstd on the other hand, we can optimize the amount of memory allocated for compression given an upper bound on the input length given each time the compressor is used - which is chunk_len_in_kb in our case.